test/main.spec.js

```diff
@@ -74,8 +75,8 @@ describe('authorizeAccessToken()', () => {
       header: {typ: 'JWT', alg: 'HS256', kid: '194B72684'},
       payload: {
         iss: 'https://issuer.example.com',
-        iat: 1613655020,
-        exp: 1645191020,
+        iat: Date.now(),
+        exp: Date.now(),
```

Comment:
These need to be in seconds since the epoch, not milliseconds.

```suggestion
        iat: Math.floor(Date.now() / 1000),
        exp: Math.floor(Date.now() / 1000),
```

Reply:
Updated here [a89942b](https://github.com/digitalbazaar/authorize-access-token-middleware/pull/3/commits/a89942b50dc9295bc6a652224008d4a8ad58a187)